### PR TITLE
build: upgrade deps for building on modern machine

### PR DESCRIPTION
**Pull request by tisonkun**
Spot when preparing 0.4.0-rc...

### Verifying this change

- [x] Make sure that the change passes the `./gradlew build` checks.

### CODE REVIEW

**Review by tuteng — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -228,12 +228,12 @@ Use the default account (`pulsar`) and the default password (`pulsar`) to log in
 
 ### Configure environment
 
-The puslar-manager supports multiple environment configurations and can manage multiple environments conveniently. 
+The puslar-manager supports multiple environment configurations and can manage multiple environments conveniently.
```
Comment (nicoloboschi) [inline README.md:231]:
```suggestion
The pulsar-manager supports multiple environment configurations and can manage multiple environments conveniently.
```

Comment (tisonkun) [reply README.md:231]: Good catch! Updated.

**Comment by tisonkun** [comment]
Pending to merge...

I'm going to prepare a 0.4.0 release candidate tomorrow.

**Review by nodece — APPROVED** [review]
LGTM

File: front-end/build.gradle [hunk 2]
```diff
@@ -18,8 +18,8 @@ plugins {
 }
 
 node {
-    version = '10.14.1'
-    npmVersion = '6.4.1'
+    version = '16.19.1'
```
Comment (nodece) [inline front-end/build.gradle:21]: Or use the 18.16 LTS version?

Comment (tisonkun) [reply front-end/build.gradle:21]: I may do it after 0.4.0 released or we find further blocker to fix :D

**Comment by tisonkun** [comment]
Merging...

**Pull request merged**